app/src/features/mocks/modals/ExportMocksModalWrapper/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from "react";
+import { useMocksModalsContext } from "features/mocks/contexts/modals";
+import { ExportMocksModal } from "./ExportMocksModal";
+
+export const ExportMocksModalWrapper: React.FC = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedMockIds, setSelectedMockIds] = useState([]);
+  const [onSuccess, setOnSuccess] = useState(() => () => {});
+
+  const { setOpenShareMocksModalAction } = useMocksModalsContext();
+
+  useEffect(() => {
+    const openModal = (selectedMockIds: string[], onSuccess?: () => void) => {
+      setIsOpen(true);
+      setSelectedMockIds(selectedMockIds);
+      setOnSuccess(() => onSuccess);
+    };
+
+    setOpenShareMocksModalAction(() => openModal);
+  }, [setOpenShareMocksModalAction]);
+
+  const onClose = () => {
+    setIsOpen(false);
+    setSelectedMockIds([]);
```

Comment:
I am hoping the [previous comment](https://github.com/requestly/requestly/pull/2123#discussion_r1740620075) was accidentally marked as resolved, since this still does not seem required.

This is building on top of the state issue I mentioned above, so IMO it would be better to not add more (_especially unnecessary_) code on top of that.